doc/ref_arch/openstack/chapters/chapter04.rst

```diff
@@ -1773,8 +1628,24 @@ may need to be made for certain Edge clouds depending upon their
 resource capabilities. It should be noted that none of these changes
 affect the definition of OpenStack flavors.
 
-Edge Cloud Deployment
-^^^^^^^^^^^^^^^^^^^^^
+The previous section listed the OpenStack services deployed on the controller nodes depepnding upon
+the control plane distribution. As specified earlier in this chapter, at least 3 controller nodes should be deployed 
+for HA.  Compute nodes may also exist at the sites where controller nodes are deployed.  
+
+Control plane services are not hosted at edge sites. Each edge site can be treated as its own OpenStack AZ. The 
+compute nodes, will host `nova-compute`, a component of the the CCompute Service (Nova), and `neutron-L2-agent`,
```

Comment:
typo: CCompute

Reply:
I don't think there is a 'neutron-L2-agent'. There is an ML2 Plugin on the controller node, but they are using a 'Linux Bridge Agent' on the Computes, see here: https://docs.openstack.org/neutron/train/install/overview.html